webservice/dispatcher.php

```diff
@@ -86,7 +86,7 @@
 // fetch the request
 WebserviceRequest::$ws_current_classname = $class_name;
 $request = call_user_func(array($class_name, 'getInstance'));
-
+$request->clearInstance();
```

Comment:
Just notice something, the clearInstance shouldn't be called here but here https://github.com/PrestaShop/PrestaShop/pull/24162/files#diff-18e4ef04b710a09044f70ca5f52bc8c1fd518a335177d776a9b3094c9b6eedfcR220

Reply:
With this call here, you're getting the instance, then directly clear it,  which has no sense